project_report/project_information.py

```diff
@@ -418,17 +459,20 @@ def find_sample_release_date_in_auth(sample):
 
     def get_library_prep_analysis_types_and_format(self):
         species = self.get_species()
-        analysis_type = self.get_analysis_type()
+        analysis_types = self.get_analysis_types()
         library_preparations = self.get_library_workflows()
-        if len(species) == 1 and species.pop() == 'Human':
-            bioinfo_analysis_types = ['bioinformatics_analysis_bcbio']
-            formats_delivered = ['fastq', 'bam', 'vcf']
-        elif analysis_type and analysis_type in ['Variant Calling', 'Variant Calling gatk']:
-            bioinfo_analysis_types = ['bioinformatics_analysis']
-            formats_delivered = ['fastq', 'bam', 'vcf']
-        else:
-            bioinfo_analysis_types = ['bioinformatics_qc']
-            formats_delivered = ['fastq']
+        bioinfo_analysis_types = set()
+        formats_delivered = set()
+        if 'Homo sapiens' in species:
+            bioinfo_analysis_types.add('bioinformatics_analysis_bcbio')
+            formats_delivered.update(['fastq', 'bam', 'vcf'])
```

Comment:
Is this line necessary, if this will be added again in line 472 or 475?

Reply:
If the analysis type UDF covers this for human (not sure if this is the case), then we can remove this

Reply:
Because we always run bcbio for human samples the "Analysis type" UDF is not always set for human samples.
Here I'm ensuring that the three format will be documented if any one samples are human or have Analysis Type said to Variant Calling.